aws/function.ts

```diff
@@ -27,6 +27,42 @@ export function createFactoryFunction(
     return new Function(name, handler, /*isFactoryFunction*/ true, opts);
 }
 
+export function createCallbackFunction(
+        name: string,
+        handler: aws.serverless.Handler | aws.serverless.HandlerFactory,
+        isFactoryFunction: boolean,
+        opts?: pulumi.ResourceOptions): aws.lambda.CallbackFunction<any, any> {
+
+    const policies = [...getComputeIAMRolePolicies()];
+    let vpcConfig: aws.serverless.FunctionOptions["vpcConfig"] | undefined;
```

Comment:
not your code, but isn't this a tautology? `T | undefined` == `T`?

Reply:
oh, I guess it's not for anything falsy... 🤦‍♂️

Reply:
yes.  yes it is.  fixing.